tests/commands/test_edit.py

```diff
@@ -0,0 +1,300 @@
+import os
+import pytest
+import subprocess
+from pathlib import Path
+
+from onyo import commands  # noqa: F401
+from onyo.commands.edit import get_editor
+
+assets = ['laptop_apple_macbookpro.0',
+          'simple/laptop_apple_macbookpro.1',
+          's p a c e s/laptop_apple_macbookpro.2',
+          's p a/c e s/laptop_apple_macbookpro.3',
+          'very/very/very/deep/laptop_apple_macbookpro.4'
+          ]
+
+
+def test_get_editor_git(repo):
+    """
+    Get the editor from git settings.
+    """
+    # set the editor
+    ret = subprocess.run(["git", "config", "onyo.core.editor", 'vi'],
+                         capture_output=True, text=True)
+    assert ret.returncode == 0
+
+    assert get_editor(repo.root) == 'vi'
+
+    # cleanup
```

Comment:
Since each test is separate, there's no need to cleanup. And indeed, it's arguably better if it is not cleaned up, as it's easier to inspect after-the-fact.

Reply:
Thanks, I adjusted it for all editor-tests.